cio/src/configs.rs

```diff
@@ -459,10 +545,31 @@ impl UserConfig {
         }
 
         // Get the Airtable information for the user.
-        let airtable_id = airtable_auth.ensure_user(db, company, &new_user).await?;
-        new_user.airtable_id = airtable_id.to_string();
-        // Update the user in the database.
-        new_user = new_user.update(db).await?;
+        match airtable_auth.ensure_user(db, company, &new_user).await {
+            Ok(airtable_id) => {
+                new_user.airtable_id = airtable_id;
+
+                // Update the user in the database.
+                new_user = new_user.update(db).await?;
+            }
+            Err(e) => {
+                warn!("Failed to ensure airtable user `{}`: {}", new_user.id, e);
+            }
+        }
+
+        // Deprovision this user explicitly from any service they should not have access to
+        for denied_service in &new_user.denied_services {
+            match denied_service.get_provider_writer(db, company).await {
+                Ok(_denied_service_provider) => info!(
+                    "User {} will be removed from {} when denied services lists are enforced",
+                    new_user.id, denied_service
+                ),
+                Err(err) => warn!(
+                    "Failed to create provider client for {} when handling denied services for user {}. err: {}",
+                    denied_service, new_user.id, err
+                ),
+            }
+        }
```

Comment:
So this isn't really able doing any deprovsioning, more just informing that a user will be deprovisioned... correct?